org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/contentassist/CompletionProposalRequestor.java

```diff
@@ -375,7 +375,7 @@ public CompletionItem toCompletionItem(CompletionProposal proposal, int index) {
 		return $;
 	}
 
-	private static Either<Range, InsertReplaceRange> getEditRange(CompletionItem completionItem, PreferenceManager preferenceManager) {
+	public static Either<Range, InsertReplaceRange> getEditRange(CompletionItem completionItem, PreferenceManager preferenceManager) {
```

Comment:
Any reason to make this public ? Doesn't look like it's used anywhere outside this class.

Reply:
Was playing around with it somewhere else but forgot to change back, fixed!